source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -279,65 +283,89 @@ def image(
 class image_features(ManagerTermBase):
     """Extracted image features from a pre-trained frozen encoder.
 
-    This method calls the :meth:`image` function to retrieve images, and then performs
-    inference on those images.
+    This term uses models from the model zoo in PyTorch and extracts features from the images.
```

Comment:
Also uses models from Transformers

Reply:
The models are in PyTorch still so this holds. Added references to later in the docs.